assemblyline/odm/models/submission.py

```diff
@@ -114,46 +120,49 @@ def create_filescore_key(self, sha256, services: list = None):
         return 'v'.join([str(hashlib.md5(s.encode()).hexdigest()), str(version)])
 
 
-@odm.model(index=True, store=True, description="Submission-Relevant Times")
+@odm.model(index=True, store=True, description="""Captures important timestamps related to the submission's lifecycle, such as when it was submitted and when the analysis was completed.
+""")
 class Times(odm.Model):
-    completed = odm.Optional(odm.Date(store=False), description="Date at which the submission finished scanning")
-    submitted = odm.Date(default="NOW", description="Date at which the submission started scanning")
+    completed = odm.Optional(odm.Date(store=False), description="Timestamp recording when the analysis of the submission was completed.")
+    submitted = odm.Date(default="NOW", description="Timestamp marking when the submission was initiated and began the analysis process.")
```

Comment:
I think we should use "recording" or "marking" in both cases, but we do not need to use two different words for the two definitions.